framework/include/timesteppers/CompositionDT.h

```diff
@@ -70,4 +120,13 @@ class CompositionDT : public TimeStepper
 
   // The time stepper(s) input as lower bound of the time stepper size
   const std::set<std::string> _lower_bound;
+
+  // The time stepper selected to use based on minimum time step size
+  TimeStepper * _last_time_stepper;
+
+  // The time stepper selected as the lower bound
+  TimeStepper * _bound_last_time_stepper;
+
+  // The time stepper selected as the time sequence stepper
+  TimeSequenceStepperBase * _last_time_sequence_stepper;
```

Comment:
same it s not really the last, more like the first

_closest_time_sequence_stepper?